k8s/helmfile/env/production/api.values.yaml.gotmpl

```diff
@@ -10,7 +10,7 @@ replicaCount:
 useProbes: true
 
 platform:
-  backendMwHost: mediawiki-138-app-backend.default.svc.cluster.local
+  backendMwHost: mediawiki-139-app-backend.default.svc.cluster.local
```

Comment:
I think things can still break in this situation yes, but I'd expect them to be less likely.
But it really depends on what is happening during the update.
As written in another comment below, really infront of all mediawiki services, (including quickstatements etc) there should be switching based on mw version so that everything always aligns on the version.